k8sinterface/k8sconfig.go

```diff
@@ -98,13 +97,18 @@ func GetK8sConfig() *restclient.Config {
 }
 
 // DEPRECATED
```

Comment:
Why are we updating this function if it is deprecated?

Reply:
I dont know who is using this function. It will probably be deleted soon.